.github/workflows/e2e-test.yaml

```diff
@@ -29,6 +29,7 @@ on:
           - linodeobj
           - linodevpc
           - linodeplacementgroup
+          - linodefirewall
```

Comment:
Not sure if you added this test to `.github/filters.yml`. We need to add it there to trigger this e2e test!

Reply:
I only see flavor tests there, none of the flavors include a firewall yet

Reply:
oh yeah we don't have firewall flavor yet. I though you added it. Sorry about that. 

e2e test you added runs under quick test so all good!